frontend/service/Autonolas.ts

```diff
@@ -134,16 +140,22 @@ const getAgentStakingRewardsInfo = async ({
 const getAvailableRewardsForEpoch = async (): Promise<number | undefined> => {
   const contractCalls = [
     serviceStakingTokenMechUsageContract.rewardsPerSecond(),
-    serviceStakingTokenMechUsageContract.livenessPeriod(),
+    serviceStakingTokenMechUsageContract.livenessPeriod(), // epoch length?
+    serviceStakingTokenMechUsageContract.tsCheckpoint(), // last checkpoint timestamp?
```

Comment:
i'm assuming the last checkpoint timestamp is roughly the same as the start of the current epoch?

Reply:
livenessPeriod is a constant for each staking program. It's the minimum epoch lenght (minimum call between checkpoints)

tsCheckpoint is the latest executed checkpoint - start of a new epoch.